tests/snowflake/test_snowflake.py

```diff
@@ -459,6 +460,25 @@ def test_oauth_refresh_token(mocker):
     assert sf.oauth_token == req_mock.return_value.json()['access_token']
 
 
+def test_schema_fields_order():
+    schema_props_keys = list(json.loads(SnowflakeConnector.schema_json())['properties'].keys())
+    assert schema_props_keys[0] == 'type'
```

Comment:
On peut tester de cette façon (peut être moins lisible ?) 
```suggestion
ordered_keys = ['type','name','account','authentication_method','user','password','oauth_token','oauth_args','role','default_warehouse','ocsp_response_cache_filename','retry_policy','secrets_storage_version','sso_credentials_keeper','user_tokens_keeper'] 
assert schema_props_keys == ordered_keys
```
ça permet de tester l'ordre

Reply:
👍